lms/static/js/learner_dashboard/RecommendationsPanel.jsx

```diff
@@ -0,0 +1,69 @@
+/* global gettext */
+import React from 'react';
+
+const RecommendationsPanel = (props) => {
+  return (
+    <div className="p-4 panel-background">
+      <div className="recommend-heading mb-4">{gettext('Recommendations for you')}</div>
+      <div className="course-card box-shadow-down-1 bg-white mb-3">
+        <div className="box-shadow-down-1 image-box">
+          <img
+            className="panel-course-img"
+            src="https://source.unsplash.com/lQGJCMY5qcM"
+            alt="course image"
+          />
+        </div>
+        <div className="course-title pl-3">The Chemistry of Life</div>
```

Comment:
Wrap this inside an anchor to make them clickable. We can update the links later on.